### DISCUSSION TITLE
The Eduverse Hackathon - August Planning

### DISCUSSION THREAD

**Discussion by mrawhite** [AngellXR/eduverse#8] [Announcements]
Hello everyone! The summer is coming to a close, and the school year is soon to start. I have reached out to various educator groups and found a few who are interested in participating in this hackathon. Exciting! I'll provide an update on things below and some action items to get the ball rolling. 

### Title
Creating the Eduverse

### Description
The pandemic has accelerated developments in virtual and augmented reality technologies, helping usher humanity into the spatial age of computing. Spatial computing provides users the ability to seamlessly interact between people, products, processes and physical spaces. Companies like [Microsoft](https://www.microsoft.com/en-us/hololens), [Facebook](https://www.oculus.com/), and [Snapchat](https://www.spectacles.com/), along with thousands of start-ups are looking to redefine how we interact with and overlay computing on the world, how we play games, and how we interact with each other. Since 2020, there has been a 1,400 percent growth in interview demand for AR/VR engineers in the past year, according to [Hired](https://hired.com/). The Eduverse seeks to introduce students to the world of augmented and virtual reality, helping to build the future of computing technology.
_(This needs a lot of work, but it's a start!)_

### Timing
The hackathon will run the entirety of the month of October.

### Curriculum
[CodeHS - Intro to VR](https://codehs.com/course/introvr/overview)
[A-Frame Docs](https://aframe.io/docs/1.2.0/introduction/)
[A-Frame School](https://aframe.io/aframe-school/)

### Platforms
Students can host their code on [Replit](https://replit.com/) or [Glitch](https://glitch.com/)

### Current Partners/Sponsors
Top of the World Alaska Computer Science Teachers Association
CodeHS
Planet Voodoo
AngellXR
**Anyone else?**

### Action Items

- [ ] Determine date for final submissions
- [ ] Determine date for judging
- [ ] Who will judge?
- [ ] Create a logo for the hackathon
- [ ] Determine prizes
- [ ] Schedule talks/meet and greets with professionals
- [ ] [Develop Eventbrite copy](https://docs.google.com/document/d/1mOu8ZZE1aGdsIKfpjcxZEJThs8vmfIxMpl-0_xtEOsM/edit?usp=sharing) (request to edit)

This is a start! Feel free to add, edit as you see fit.

**Comment by Thiizic** [comment]
Not sure where to put this,

Futurist Foundation can pledge free hosting for the website and a Wordpress site.
Potentially prizes or other funding.

We also have access to a nonprofit account on Canva that we can use for this.

Additionally we have access to STEM based subreddits on reddit.
